docs/orchestrating-elastic-stack-applications/rotate-credentials.asciidoc

```diff
@@ -0,0 +1,33 @@
+:page_id: rotate-credentials
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Rotate auto-generated credentials
+
+When deploying an Elastic Stack application, the operator generates a set of credentials essential for the operation of that application. Examples of these generated credentials include the default `elastic` user for Elasticsearch and the security token for APM Server. You can list all auto-generated credentials in a namespace by running the following command:
+
+[source,sh]
+----
+kubectl get secret -l eck.k8s.elastic.co/credentials=true
+----
+
+You can force the auto-generated credentials to be regenerated with new values by simply deleting the appropriate Secret. For example, to change the password for the `elastic` user from the <<{p}-quickstart,quickstart example>>, execute the following command:
+
+[source,sh]
+----
+kubectl delete secret quickstart-es-elastic-user
+----
+
+CAUTION: If you are using the `elastic` user credentials in your own applications, they will fail to connect to Elasticsearch and Kibana after the above step. It is not recommended to use `elastic` user credentials for production use cases. Always <<{p}-users-and-roles,create your own users with restricted roles>> to access Elasticsearch.
```

Comment:
> It is not recommended to use `elastic` user credentials for production use cases.

I don't know about that part. Why isn't it recommended?

Reply:
Because it's a superuser?